_data/StemsAnalyzer.yml

```diff
@@ -0,0 +1,9 @@
+description: An extension for precision point insertion.
+developer: Jérémie Hornus and Connor Davenport
+developerURL: https://black-foundry.com https://connordavenport.com
+extensionName: StemsAnalyser
+extensionPath: StemsAnalyser.roboFontExt
+repository: https://github.com/connordavenport/RobofontTools/tree/master/StemsAnalyser/StemsAnalyzer.roboFontExt
```

Comment:
I would prefer if extensions from a big repo are split up into smaller repos per extension.

the download would be bigger, checking versions takes longer... 

I can also host abandoned but super usefull extension on [robodocs](https://github.com/roboDocs) and make you admin of that repo